src/components/input/input-default.jsx

```diff
@@ -99,37 +99,41 @@ class InputDefault extends React.PureComponent {
     }
   }
 
-  renderInput(id, type) {
+  renderInput(id) {
     const classes = `input__element input__element--${this.props.type}`;
     const placeholder = this.props.placeholder || this.props.label;
 
-    return (
-      <input
-        {...omit(
-          this.props,
-          'valueFormatter',
-          'hasSuccess',
-          'hasWarning',
-          'hasError',
-          'helpText',
-          'leftAddon',
-          'rightAddon',
-          'options',
-          'classes',
-          'sheet',
-          'theme',
-          'variant',
-        )}
-        id={id}
-        className={classes}
-        type={type || this.props.type}
-        onFocus={this.onFocus}
-        onBlur={this.onBlur}
-        onChange={this.onChange}
-        placeholder={placeholder}
-        value={this.state.value}
-      />
-    );
+    const props = {
+      ...omit(
+        this.props,
+        'valueFormatter',
+        'hasSuccess',
+        'hasWarning',
+        'hasError',
+        'helpText',
+        'leftAddon',
+        'rightAddon',
+        'options',
+        'classes',
+        'sheet',
+        'theme',
+        'variant',
+        'lineCount',
+      ),
+      id,
+      placeholder,
+      className: classes,
+      onFocus: this.onFocus,
+      onBlur: this.onBlur,
+      onChange: this.onChange,
+      value: this.state.value,
+    };
+
+    if (this.props.type === 'textarea') {
+      return <textarea {...props} style={{ height: this.props.lineCount * 21 }} />;
```

Comment:
This style can be put in the class like so:
```
...: {
height: ({ lineCount }) => lineCount * 21,
}
```

Also, we might want to consider using something else than pixels so that fontSize can vary.

Reply:
@bstream nice. fixed on both counts